apps/monitoring/helm-base-versions/helm-base-versions.yaml

```diff
@@ -30,17 +30,19 @@ spec:
     imagePullPolicy: Always
     environment:
       CLUSTER_NAME: "cft-${CLUSTER_FULL_NAME}-aks"
-      SLACK_WEBHOOK:
-        valueFrom:
-          secretKeyRef:
-            name: helm-base-versions
-            key: slack-webhook
       ARTIFACT_URL: "http://source-controller.flux-system.svc.cluster.local."
       API_SERVER_URL: "https://kubernetes.default.svc.cluster.local"
+    secrets:
+      COSMOS_KEY:
+        secretRef: helm-base-versions
+        key: cosmos-key
```

Comment:
This one isn't necessary, it wasnt in the original config and it should be using Managed Identities now for access.